ONSdigital/ssdc-rm-support-tool
# 3246 Personalisation on SMS and email fulfilments

# Motivation and Context

SMS and email fulfilments will now support added personalisation values on the request, so the support tool UI needs to support it.

# What has changed


-
- Add dynamic personalisation form section to SMS and email fulfilment request forms 
- Pass personalisation through to notify request in the back end
- Factor out personalisation form as component
- Sort out template/packCode naming confusion in JS

# How to test?



Build and run the linked branches, create SMS and email templates with and without `__request__.` prefixed personalisation fields in the template. When you request an SMS/email fulfilment on the case, when selecting a template which includes these fields, the form should update to include those optional personalisation fields.

# Links



https://trello.com/c/8DGOfdJx/3246-personalisation-on-sms-and-email-fulfilments-8
